app/views/home/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<h1>Home#index</h1>
+<p>Find me in app/views/home/index.html.erb</p>
+<!-- app/views/pages/index.html.erb -->
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Your App</title>
+</head>
+<body>
+  <h1>Welcome to Your App</h1>
+  <p>Click the button below to get a random greeting:</p>
+  <button id="getRandomGreeting">Get Random Greeting</button>
+  <p id="greeting"></p>
+  
+  <script>
+    document.getElementById('getRandomGreeting').addEventListener('click', () => {
+      fetch('/api/greetings/random')
+        .then(response => response.json())
+        .then(data => {
+          document.getElementById('greeting').innerText = data.greeting;
+        })
+        .catch(error => console.error('Error:', error));
+    });
+  </script>
+</body>
```

Comment:
- Good job done so far, but according to the project requirements, this should be handled using `react/redux` and it should be inside the `app/javascript/application.js` file. Please use `React and Redux` to fetch the data on the frontend 👍 👍